prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -221,7 +230,7 @@ public async Task<ActionResult> SetEnrolleePaperSubmissionProfileCompleted(int e
         /// </summary>
         /// <param name="enrolleeId"></param>
         [HttpGet("{enrolleeId}/paper-submissions/documents", Name = nameof(GetAdjudicationDocuments))]
-        [Authorize(Roles = Roles.ViewEnrollee)]
+        [Authorize(Roles = Roles.TriageEnrollee + "," + Roles.ViewEnrollee)]
```

Comment:
I think this can just be `Roles.ViewEnrollee`.

Reply:
For reference:
```c#
[Authorize(Roles = Roles.TriageEnrollee)]
[Authorize(Roles = Roles.ViewEnrollee)]
```
Requires both headers to resolve, while
```c#
[Authorize(Roles = Roles.TriageEnrollee + "," + Roles.ViewEnrollee)]
```
means any one of them will be accepted.

Reply:
Fixed.